_includes/header/header-id.html

```diff
@@ -122,7 +122,10 @@
                               <a href="/{{ page.lang }}/resources/contributing.html">Contributing to Express</a>
                             </li>
                             <li>
-                              <a href="/{{ page.lang }}/changelog/4x.html">Release Change Log</a>
+                              <a href="/en/resources/contributing-docs.html">Contributing to Docs</a>
+                            </li>
+                            <li>
+                              <a href="/en/changelog/4x.html">Release Change Log</a>
```

Comment:
This page was broken in Indonesian. Added link 
![id_changelog](https://github.com/user-attachments/assets/900de271-c643-4bfa-8fa7-c302a0c2d90b)